management_interface/management_interface/forms.py

```diff
@@ -19,9 +25,46 @@ class Meta:
 
 
 class CareRecipientForm(forms.ModelForm):
+    given_name = forms.CharField(max_length=64, required=True, label="Given Name")
```

Comment:
Change to reflect needing to enter all given names

```suggestion
    given_name = forms.CharField(max_length=64, required=True, label="Given Names", help_text="enter all given (first and middle) names, separated by spaces, e.g. Lavina Maxine")
```